unstructured_ingest/v2/processes/connectors/weaviate/weaviate.py

```diff
@@ -205,17 +208,54 @@ def get_batch_client(self, client: "WeaviateClient") -> Generator["BatchClient",
 
 
 @dataclass
-class WeaviateUploader(Uploader, ABC):
+class WeaviateUploader(VectorDBUploader, ABC):
     upload_config: WeaviateUploaderConfig
     connection_config: WeaviateConnectionConfig
 
+    def _existing_collections(self) -> list[str]:
```

Comment:
I'm assuming there is no api to just get a single collection to see if it exists?